cli/lsp/language_server.rs

```diff
@@ -2126,13 +2126,20 @@ impl Inner {
     }
   }
 
-  async fn request_else(
+  // NOTE(CGQAQ): tower-lsp don't have [request_else](https://docs.rs/lspower/1.5.0/src/lspower/lib.rs.html#758)
```

Comment:
This note does not need to exist in the code... once migrated, no one will care.

Reply:
ok :)